ci/report-tests.sh

```diff
@@ -0,0 +1,149 @@
+            sed -i -e 's?/bower_components/twentytwenty/js/jquery.twentytwenty.js?https://cdnjs.cloudflare.com/ajax/libs/mhayes-twentytwenty/1.0.0/js/jquery.twentytwenty.min.js?g'   "${TEST_RESULTS_FILE}"
+            sed -i -e 's?/bower_components/clipboard/dist/clipboard.min.js?https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.5.16/clipboard.min.js?g'                             "${TEST_RESULTS_FILE}"
+            sed -i -e 's?/bower_components/anchor-js/anchor.min.js?https://cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.2/anchor.min.js?g'                                            "${TEST_RESULTS_FILE}"
+            sed -i -e 's?/bower_components/featherlight/release/featherlight.min.js?https://cdnjs.cloudflare.com/ajax/libs/featherlight/1.7.13/featherlight.min.js?g'                 "${TEST_RESULTS_FILE}"
+            sed -i -e 's?/bower_components/bootstrap/dist/js/bootstrap.min.js?https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js?g'                   "${TEST_RESULTS_FILE}"
+            sed -i -e 's?/bower_components/dustjs-linkedin/dist/dust-full.min.js?https://cdnjs.cloudflare.com/ajax/libs/dustjs-linkedin/2.7.5/dust-full.min.js?g'                     "${TEST_RESULTS_FILE}"
+            sed -i -e 's?/bower_components/numeral/min/numeral.min.js?https://cdnjs.cloudflare.com/ajax/libs/numeral.js/2.0.4/numeral.min.js?g'                                       "${TEST_RESULTS_FILE}"
+
+        else
+            echo "The Unreal Editor crashed while running tests, see the test-gdk annotation for logs (or the tests.log buildkite artifact)."
+            exit 1
+        fi
+
+        # Upload artifacts to Buildkite, capture output to extract artifact ID in the Slack message generation.
+
+        buildkite-agent artifact upload "index.json"
+        if [[ $? -ne 0 ]]; then
+            echo "Failed to upload artifact index.json."
+            exit 1
+        fi
+        UPLOAD_OUTPUT=$(buildkite-agent artifact upload "index.html" 2>&1 > /dev/null)
+        if [[ $? -ne 0 ]]; then
+            echo "Failed to upload artifact index.html."
+            exit 1
+        fi
+
+        # Artifacts are assigned an ID upon upload, so grab IDs from upload process output to build the artifact URLs
+        # The output log is: "Uploading artifact <artifact id> <upload path>". We are interested in the artifact id
+        REGEX='Uploading artifact ([a-z0-9-]*) .*index\.html'
+        if [[ ${UPLOAD_OUTPUT} =~ ${REGEX} ]]; then
+            TEST_RESULTS_URL="${UPLOAD_ARTIFACT_PATH}/${BASH_REMATCH[1]}"
+        else
+            echo "Failed to extract artifact ids"
+            exit 1
+        fi
+
+        echo "Test results in a nicer format can be found <a href=${TEST_RESULTS_URL}>here</a>."| buildkite-agent annotate \
+            --context "unreal-gdk-test-artifact-location" \
+            --style info
+
+        # Read the test results
+        RESULTS_PATH="index.json"
```

Comment:
I think since you define this var down here you can afford to use it above too where there are usages of the string `index.json`.
Perhaps it's worth doing this with the `index.html` too